examples/v3/provider-state-injected/package.json

```diff
@@ -11,9 +11,9 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@pact-foundation/pact": "10.1.2",
+    "@pact-foundation/pact": "11.0.2",
     "chai": "^4.3.6",
-    "jest": "^26.6.3"
```

Comment:
jest 26 fails on the node:path lookup suggested and implemented  here

https://github.com/pact-foundation/pact-js-core/pull/446#discussion_r1193067918

updating to jest 29 resolves it, didn't test other versions in between